### DISCUSSION TITLE
Recommend some good English quotes

### DISCUSSION THREAD

**Discussion by zztqsh** [zztqsh/THEO1126.github.io#3] [General]
Recommend some good English quotes

**Comment by THEO1126** [comment]
> Recommend some good English quotes

The world may be full of cheating, however wenever lack friends with a warm heart.

**Reply by zztqsh** [reply]
The world may be full of cheating, however wenever lack friends with a warm heart.

**Reply by THEO1126** [reply]
The world may be full of cheating, however wenever lack friends with a warm heart.

**Comment by zztqsh** [comment]
Recommend some good English quotes

**Reply by THEO1126** [reply]
Recommend some good English quotes

**Comment by THEO1126** [comment]
The world may be full of cheating, however wenever lack friends with a warm heart.
